src/main/java/com/bandwidth/sqs/queue/buffer/KeyedTaskBuffer.java

```diff
@@ -72,7 +68,11 @@ public TimerExpiredTask(K key) {
 
         @Override
         public void run() {
-            processBatchIfNeeded(key, true);
+            try {
+                processBatchIfNeeded(key, true);
+            } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
Does this library not use SLF4J? Why not `LOG.error`

Reply:
I'll add a task to use a logging framework, but right now this will give us what we need